test/globals.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+global.chai = require('chai')
+global.should = global.chai.should()
+global.sinon = require('sinon')
+global.proxyquire = require('proxyquire')
```

Comment:
do we even want these to be global? Why dont we just require them when we need them?